### ISSUE TITLE
Summer Shape Up! 4 Alternative Weight Loss Solutions

### ISSUE THREAD

**Issue by nadjibandnaaima** [nadjibandnaaima/diets-for-quick-weight-loss#561]
<div dir="ltr" style="text-align: left;">I am embarrassed to admit how many different commercial weight loss programs I've tried. I'm sure you'll nod in agreement when I tell you I've grown tired of talking heads touting empty promises. None of them helped me look the way I think I should and most of them left me agonizing over eating the same boring foods over and over. If you've been in the game for awhile, you already know the basics of exercise: train with resistance and train with cardio. When it comes to food though, what's a body to do?<br><br>
If you've been frustrated by commercial dieting take heart: In this article you'll find a few not-so-commercial shape up tidbits. Ones that they don't always tell you. Ones that I've found really work. So whether you need to lose a few pounds left over from winter or are looking for a healthy lifestyle change and are tired of the same-old routines, these four alternative weight loss solutions will help get you started.<br><br>
Solution #1: Use Creative Visualization<br><br>
Stimulating the mind to help your body achieve a desired result is a great way to bring about change. "Seeing" yourself thinner and imposing the mind with positive thoughts is one of the most successful ways to decrease your body size and improve body image. Known often as guided imagery, it works by training your mind to stimulate changes in your body. This can be accomplished by using some of my favorite tools:<br><br>
Audio CD's: If you are an auditory person, you will love the guided imagery and weight loss CD available from Donate Your Weight: http://ift.tt/1J0BEjl. This CD is a combination of creative visualization, hypnosis and body affirming exercises. Join your host hypnotherapist Sheri Zampelli as she leads you though several positive body changing exercises. I use this CD often and love it. Not only does it stop negative body-image feelings in their tracks, but as a bonus the listener also receives a feeling of relaxation that lasts throughout the day.<br><br>
Writing Affirmations: Using affirmations will train you mind by stating or affirming your intentions. It will retrain your thinking toward positive thoughts. When I use affirmations, I like to use positive statements such as these: My Body Is Beautiful! I Love My Body! To increase their effectiveness, I write these messages to myself on an index card or piece of paper and carry them with me. I review them several times throughout the day.<br><br>
Books: One of my favorite positive affirmation books is Love Your Body! By Louise L. Hay available from www.hayhouse.com. Inside this book, Louise has created affirmations for you . If you read through them each day you are sure to reap the rewards of loving your body.<br><br>
Solution #2 Try An Herbal Wrap<br><br>
If you need to lose a few extra pounds, you may want to try an herbal wrap. Herbal wraps help the body lose water and inches. You can get a wrap from your favorite salon, however, they tend to be expensive. It's just as easy to give yourself a spa treatment at home. Herbal wrap products are everywhere. If you'd like to get a wrap but prefer the privacy of your own home. check out www.slenderwrap.com<br><br>
Some of the benefits of herbal wraps include:<br><br>
- Weight loss<br>
- Water loss<br>
- Decreased inches<br>
- Smoothness of tissue<br><br>
Solution #3 Stop When You Are Full<br><br>
In her book Appetites, Geneen Roth gives us a simple outline to manage portion control. Her two major mantras are: take time to eat and stop when you are full. These are both shape-shifting ideas. To some people, including myself, food is a friend. Therefore the tendency to overeat stems out of emotion: food provides comfort and security. Others are in a hurry. Food and the need to eat are just obstacles that get in the way.<br><br>
In effort to break free from these unhealthy behaviors, ask yourself several questions:<br><br>
Am I enjoying what I am eating? Do I eat standing up or on the run? Do I continue to eat even after I know I am full? Next, begin to carve out time to eat and enjoy food with friends or family. Or to take time to enjoy a nourishing meal even if you are alone. Eat to the point of being full, then stop. Most of the time, weight loss can be easily stimulated through portion control.<br><br>
Solution #4 Keep A Food Journal<br><br>
In addition to the aforementioned, it may be wise for you to keep a food journal. By tracking what you eat every day, you hold yourself accountable for everything you place in your mouth. it's simple to start. Record everything you eat for a week and document your weight.<br><br>
At the end of the week take inventory of yourself. have you gained weight? Lost? Remained the same? If you are gaining weight, simply cut back on your refined sugars (sweets) and heavy carbs like white rice or white potatoes. For these, substitute brown rice or sweet potatoes instead. Track your habits for a second week.<br><br>
When you've reached the point in your eating program where your weight remains the same, you will need only consume a few less calories (Or burn a few extra during exercise) each week to lose weight. Keep in mind: in order to lose weight, your body needs to burn more calories than it consumes. Remember: First find you melting point, then adjust your portion control and exercise program accordingly.<br><br>
In Conclusion: With just a few adjustments and an enjoyment of food by recognizing its beauty for nourishing your body and a conscious change in body image awareness, you can shift your focus from frustration to enjoyment. Give these alternatives to weight loss a try. Your health and your waist line will thank you for it.<br><br><br>
</div>
<br><br>
via diets for quick weight loss http://ift.tt/1T8lqVJ<br>
December 19, 2015 at 04:09PM
